include/inviwo/core/io/serialization/serializebase.h

```diff
@@ -192,7 +192,7 @@ decltype(auto) toStr(const T& value) {
     } else {
         std::ostringstream stream;
         if constexpr (std::is_same_v<T, double>) {
-            stream.precision(40);
+            stream.precision(17);
```

Comment:
Why 17? (Could say the same about 40...) There should be a comment here

Reply:
thought 16 would be enough for a double, but then one of the unit tests fail to roundtrip a double max. 17 works fine. 40 is just crazy :)